docs/credits.md

```diff
@@ -982,6 +983,7 @@ Octopus Deploy is made possible thanks to the following great third-party produc
 | through2                                          | Rod Vagg                                                                  | [https://github.com/rvagg/through2](https://github.com/rvagg/through2)                                                                                                                                                      | [MIT](https://opensource.org/licenses/MIT)                                                                          |
 | thunky                                            | Mathias Buus Madsen                                                       | [https://github.com/mafintosh/thunky](https://github.com/mafintosh/thunky)                                                                                                                                                  | [MIT](https://opensource.org/licenses/MIT)                                                                          |
 | timers-browserify                                 | J. Ryan Stinnett                                                          | [https://github.com/jryans/timers-browserify](https://github.com/jryans/timers-browserify)                                                                                                                                  | [MIT](https://opensource.org/licenses/MIT)                                                                          |
+| TimeZoneConverter                                 | Matt Johnson                                                              | [https://github.com/mj1856/TimeZoneConverter](https://github.com/mj1856/TimeZoneConverter)                                                                                                                                  | [Undefined]()                                                                                                       |
```

Comment:
This is a bit interesting @matt-richardson - it's actually MIT, but this says undefined.

Reply:
Its because he's [using](https://github.com/mj1856/TimeZoneConverter/blob/master/src/TimeZoneConverter/TimeZoneConverter.csproj#L10) the `PackageLicense` tag, which nuget has [stuffed around with right royally](https://github.com/NuGet/Home/issues/7509). Also, we're using a really old nuget, as upgrading to a new version of nuget is a _nightmare_. Think we'll need to tackle it though :(

Reply:
Manually tweaked, to give me a weeks grace